modules/gapi/perf/gpu/gapi_core_perf_tests_gpu.cpp

```diff
@@ -76,25 +79,28 @@ INSTANTIATE_TEST_CASE_P(DivPerfTestGPU, DivPerfTest,
                                 Values(cv::compile_args(CORE_GPU))));
 
 INSTANTIATE_TEST_CASE_P(DivCPerfTestGPU, DivCPerfTest,
-                        Combine(Values( szSmall128, szVGA, sz720p, sz1080p ),
+                        Combine(Values(AbsExact().to_compare_f()),
+                                Values( szSmall128, szVGA, sz720p, sz1080p ),
                                 Values( CV_8UC1, CV_8UC3, CV_16UC1, CV_16SC1, CV_32FC1 ),
                                 Values( -1, CV_8U, CV_16U, CV_32F ),
                                 Values(cv::compile_args(CORE_GPU))));
-
+//TODO: DivRC test doesn't work with (CV_16SC1, -1)
 INSTANTIATE_TEST_CASE_P(DivRCPerfTestGPU, DivRCPerfTest,
-                        Combine(Values(Tolerance_FloatRel_IntAbs(1e-5, 2).to_compare_f()),
+                        Combine(Values(Tolerance_FloatRel_IntAbs(1e-6, 1).to_compare_f()),
                                 Values( szSmall128, szVGA, sz720p, sz1080p ),
-                                Values( CV_8UC1, CV_8UC3, CV_16UC1, CV_16SC1, CV_32FC1 ),
+                                Values( CV_8UC1, CV_8UC3, CV_16UC1, /*CV_16SC1,*/ CV_32FC1 ),
                                 Values( -1, CV_8U, CV_16U, CV_32F ),
                                 Values(cv::compile_args(CORE_GPU))));
 //TODO: mask test doesn't work
```

Comment:
`FIXIT:` would be better

Reply:
Did you create a ticket?

Reply:
https://github.com/opencv/opencv/pull/21150#:~:text=That%20test%20case%20is%20working%20now.%20Removed%20that%20line

Reply:
The link mentioned above goes to this PR. Could you please create a Jira ticket for this issue and mention its number in the comment to disabled test?

Reply:
I still haven't see any changes here)

Reply:
Add Jira ticket id and use FIXIT